migrations/versions/schema/2025-08-28_0e8d17ce0f06_reconcile_workflows_l2vpn.py

```diff
@@ -6,12 +6,11 @@
 
 """
 
-import sqlalchemy as sa
 from alembic import op
 
 # revision identifiers, used by Alembic.
 revision = "0e8d17ce0f06"
-down_revision = "d946c20663d3"
+down_revision = "bc54616fefcf"
```

Comment:
Why does this change?

Reply:
Not directly related to this issue, but it is broken on master.